components/brave_wallet/browser/brave_wallet_constants.cc

```diff
@@ -99,4 +100,49 @@ const base::flat_map<std::string, std::string>& GetAnkrBlockchains() {
 
   return *blockchains;
 }
+
+// CSP Override for CSPDirectiveName::FrameSrc.
+// On desktop, will return:
+//   "frame-src chrome-untrusted://nft-display/
+//   chrome-untrusted://line-chart-display/ chrome-untrusted://market-display/
+//   chrome-untrusted://trezor-bridge/ chrome-untrusted://ledger-bridge/;"
+// On mobile, will return:
+//   "frame-src chrome-untrusted://nft-display/
+//   chrome-untrusted://line-chart-display/ chrome-untrusted://market-display/;"
+std::string GetWalletFrameSrcCSP() {
```

Comment:
why are these in constants? There are some other methods already in here that don't make sense in a constants.h file, but I would put these in some kind of helper specific to the relevant webui.